default/web_tt2/sso_login.tt2

```diff
@@ -1,35 +1,36 @@
 <!-- sso_login.tt2 -->
 [% IF subaction == 'requestemail' %]
+    [%|loc%]The mailing list server requires a working email address. Please provide your email address below. <p><i>Please note:</i> the email field may be pre-populated.  If it is, then your authentication server has supplied an email address it associates with you.  Whether you enter an address manually or accept the one provided, you will still need to confirm that this is a working email address by entering a confirmation password that will be sent to you.  Please press the "Send me a confirmation password" to continue to the confirmation step.[%END%]
+    <br />
+    <p>
+        [%|loc%]A confirmation password will be sent to the address you supply, after pressing the button. This will take you to the next screen to confirm that your address is operational.[%END%]
+    </p>
```

Comment:
I think these paragraphs should be:
``` code
    <p>
        [%|loc%]The mailing list server requires a working email address. Please provide your email address below.[%END%]
    </p>
    <p>
        <i>[%|loc%]Please note:[%END%]</i>
        [%|loc%]the email field may be pre-populated.  If it is, then your authentication server has supplied an email address it associates with you.  Whether you enter an address manually or accept the one provided, you will still need to confirm that this is a working email address by entering a confirmation password that will be sent to you.  Please press the "Send me a confirmation password" to continue to the confirmation step.[%END%]
    </p>
    <p>
        [%|loc%]A confirmation password will be sent to the address you supply, after pressing the button. This will take you to the next screen to confirm that your address is operational.[%END%]
    </p>
```
Note: Message catalog should be tweaked

Reply:
As it beyond indentation fix, we may put that change in another issue, don't you think?

Reply:
I agree, and submitted PR #592.